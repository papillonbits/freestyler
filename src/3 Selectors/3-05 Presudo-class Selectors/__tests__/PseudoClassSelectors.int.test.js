import { regular } from './PseudoClassSelectors.int.story'

describe('<PseudoClassSelectors />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
