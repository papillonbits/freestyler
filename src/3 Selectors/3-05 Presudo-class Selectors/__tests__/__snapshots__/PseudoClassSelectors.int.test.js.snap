// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PseudoClassSelectors /> Render must match regular() 1`] = `
<div>
  <div
    class="pseudo-class-selectors"
  >
    <h1>
      Pseudo-class Selectors
    </h1>
    <p>
      <a
        href="http://www.cssdesignpatterns.com"
      >
        a:link -- unvisited link
      </a>
      <a
        href="http://www.htmldesignpatterns.com"
      >
        a:visited -- visited link
      </a>
      <a
        href="http://www.cssdesignpatterns.com"
      >
        a:hover -- mouse hovering
      </a>
      <a
        href="http://www.cssdesignpatterns.com"
      >
        a:active -- visiting a link
      </a>
    </p>
  </div>
</div>
`;
